playground/vite.config.ts

```diff
@@ -23,5 +27,11 @@ export default defineConfig({
   build: {
     target: 'esnext',
     assetsInlineLimit: 0,
+    rollupOptions: {
+      input: {
+        appSchoool: fileURLToPath(new URL('./index.html', import.meta.url)),
+        appStudent: fileURLToPath(new URL('./test.html', import.meta.url)),
```

Comment:
Is the school and student naming convention a playwright thing?

Reply:
😂 leftover from an example, I'll give them proper names